squeasel.c

```diff
@@ -4224,12 +4231,10 @@ static int set_ssl_option(struct sq_context *ctx) {
   int options = SSL_OP_NO_SSLv2 | SSL_OP_NO_SSLv3;
   if (sq_strcasecmp(ssl_version, "tlsv1") == 0) {
     // No-op - don't exclude any TLS protocols.
-#if OPENSSL_VERSION_NUMBER >= OPENSSL_VERSION_HAS_TLS_1_1
```

Comment:
maybe I'm missing something, but I thought the point here was to always compile support even if we are building against an older OpenSSL? checking OPENSSL_VERSION_NUMBER is a compile-time check so it would compile out the support if building against old SSL?

Reply:
The check is removed (going from my patch -> cloudera/master) - am I missing something?

Reply:
oh, duh. I am used to the "split view" from gerrit but github defaults to 'unified' view, so when I saw this I thought it was added code. sorry for the noise